packages/atomic/src/components/atomic-context-provider/atomic-context-provider.tsx

```diff
@@ -0,0 +1,19 @@
+import {Component, Prop} from '@stencil/core';
```

Comment:
I added this atomic component to test context, I can keep it or remove it. I'm not sure if it make sense with our current design for atomic because it has no actual UI layer.

Reply:
Hmm, it would depend on how a developer will interact with it. The component makes it easy to define static context. However, we also need to consider how a developer would be able to dynamically set context.

We can keep the component for now, and adjust depending on the answer